orca-clouddriver/src/main/java/com/netflix/spinnaker/orca/clouddriver/tasks/cache/ClouddriverClearAltTablespaceTask.java

```diff
@@ -48,18 +47,21 @@ public TaskResult execute(@Nonnull StageExecution stage) {
           result.getOrDefault("tables", emptyList()));
 
       return TaskResult.builder(ExecutionStatus.SUCCEEDED).context(result).build();
-    } catch (RetrofitError e) {
-      Map<String, Object> output = new HashMap<>();
-      List<String> errors = new ArrayList<>();
+    } catch (SpinnakerHttpException e) {
+      output = new HashMap<>();
 
-      if (e.getResponse() != null && e.getResponse().getBody() != null) {
-        String error = new String(((TypedByteArray) e.getResponse().getBody()).getBytes());
-        log.error("Failed clearing clouddriver table namespace: {}", error, e);
-        errors.add(error);
+      if (e.getResponseBody() != null) {
+        Map<String, Object> errorBody = e.getResponseBody();
+        log.error("Failed clearing clouddriver table namespace: {}", errorBody, e);
+        output.put("errors", errorBody);
       } else {
-        errors.add(e.getMessage());
+        output.put("errors", e.getMessage());
       }
-      output.put("errors", errors);
+      return TaskResult.builder(ExecutionStatus.TERMINAL).context(output).build();
+    } catch (SpinnakerServerException e) {
```

Comment:
instanceof is never great, but in this case having one `catch (SpinnakerServerException e) {` block and handling the SpinnakerHttpException case within it is I think overall simpler.  No need to declare outputs above, and we reduce some duplication.

Reply:
Removed the catch block which handles SpinnakerHttpException and now it has just one catch block to handle the SpinnakerServerException which will now simply populates the Task context property with the exception message.